woocommerce/wc-admin
# Remove coupon_total from order stats table

Fixes #1942 

* Removes the `coupon_total` from the order stats table.
* Replaces it with summed data from the join already in place with the coupons table.
* Adds coupon data to tests.

### Detailed test instructions:

1.  Create an order with coupons.
2.  Check that the coupon amount shown in the Revenue reports is accurate.
3.  Check that all tests pass. ✅
